docs/src/guides/setup-dev.md

```diff
@@ -1,68 +1,21 @@
-# Installing dependencies
+# Installing development setup for ZKsync
 
 ## TL;DR
 
-This is a shorter version of setup guide to make it easier subsequent initializations. If it's the first time you're
+This is a shorter version of setup guide to make it easier for initializations. If it's the first time you're
 initializing the workspace, it's recommended that you read the whole guide below, as it provides more context and tips.
 
-If you run on 'clean' Ubuntu on GCP:
+Just run the following command on Debian-based Linux:
```

Comment:
I am also hesitatnt to suggest blindly running it. If the script will be run on a machine where, e.g., docker is already installed (_especially_ if it was installed in some other way), it can mess up the configuration.

Do we really need to have a script here? If so, I'd probably want to make it fail-proof, e.g. that it won't do stuff that is not needed, but it's a much bigger effort.

The benefit of a code block is that you see the code and can choose what lines you need and what lines you don't need.

Reply:
If we are to keep this, I'd add some big red warning that you should first read the script code and make sure that it does what you need (which though would render it much less helpful).

Reply:
Fair enough, but the original setup guide specifically says that the code is to be run on a fresh GCP instance. So it seems reasonable that we have a fast setup option for those cases at least.
I can add a code block below with what's in the script, so that people can inspect it though.

Reply:
Original -- yes, the new one removed mention of GCP.
Regarding code block -- it would mean duplication, and script will likely get out of sync with the code block.

Reply:
Then the only possible solution is for me to close this PR.

Reply:
> I don't really think it's possible to create a fail-proof version of this script (at least without a disproportionate amount of work).

Yup, I agree, and that's exactly the reason why it wasn't done before.

> And if we don't want duplication then the only possible solution is for me to close this PR.

Was there any reason why we need a script except for making usage simpler?

Reply:
No, just to make usage simpler. I created one for myself, since I have to run fresh dev nodes with some frequency, and thought it would be useful for more people.